src/storage/src/sst/parquet.rs

```diff
@@ -464,7 +445,7 @@ impl ArrowPredicate for PlainTimestampRowFilter {
     }
 
     fn evaluate(&mut self, batch: RecordBatch) -> std::result::Result<BooleanArray, ArrowError> {
-        let ts_col = batch.column(self.timestamp_index);
+        let ts_col = batch.column(0);
```

Comment:
Do we provide any guarantee that the first column is timestamp index column?

Reply:
We've passed a projection with only timestamp column to build predicate, when it's evaluated only the timestamp column will be passed thru.